### PR TITLE
Make sure to checkout the correct ezplatform branch to run behat tests

### PR DESCRIPTION
**Pull request by dpobel**
See https://github.com/ezsystems/PlatformUIBundle/pull/755

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,6 +32,8 @@
         }
     },
     "extra": {
+        "_ezplatform_branch_for_behat_tests_comment_": "ezplatform branch to use to run Behat tests",
+        "_ezplatform_branch_for_behat_tests": "1.6",
```
Comment (andrerom) [inline composer.json]: as this it toward master vranch it should be master here

Comment (dpobel) [inline composer.json]: wrong base branch actually :)

Comment (andrerom) [inline composer.json]: ;)

File: bin/.travis/prepare_behat.sh [hunk 2]
```diff
@@ -6,10 +6,13 @@
 git fetch --unshallow && git checkout -b tmp_travis_branch
 export BRANCH_BUILD_DIR=$TRAVIS_BUILD_DIR
 export TRAVIS_BUILD_DIR="$HOME/build/ezplatform"
+
+# Checkout meta repo, use the branch indicated in composer.json under extra._ezplatform_branch_for_behat_tests
+EZPLATFORM_BRANCH=`php -r '$composer = json_decode(file_get_contents("./composer.json")); echo $composer->extra->_ezplatform_branch_for_behat_tests;'`
```
Comment (bdunogier) [inline bin/.travis/prepare_behat.sh]: Same remark than on the PlatformUI PR (shorter PHP).

**Pull request merged**